### PR TITLE
feat: add chromecast support

### PR DESCRIPTION
**Pull request by Bitaru**
Adding support for Chromecast into the player

Closes https://github.com/Uscreen-video/video-player/issues/4

### CODE REVIEW

File: src/components/video-chromecast/Video-chromecast.component.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    try {
+      await window.cast.framework.CastContext.getInstance().requestSession()
+
+      const session = window.cast.framework.CastContext.getInstance().getCurrentSession()
+
+      await session.loadMedia(request)
+
+      this.targetDevise = this.player?.statusText || ''
+    } catch (err) {
+    }
```
Comment (timaramazanov) [inline src/components/video-chromecast/Video-chromecast.component.ts:150]: should we at least log smth here ?

**Review by timaramazanov — APPROVED** [review]
✅ Looks good

Overall looks good, but mb we can create separate module for chromecast API ? E.g. adapter for chromecats api so we don't use it directly in the player code, so it would be a bit easier to refactor

**Comment by Bitaru** [comment]
> ✅ Looks good
> 
> Overall looks good, but mb we can create separate module for chromecast API ? E.g. adapter for chromecats api so we don't use it directly in the player code, so it would be a bit easier to refactor . E.g. all code realated to the `window.chrome.cast`

Don't think its necessary, there are just couple of methods that belong to the cast directly.

**Pull request merged**